introspection-engine/connectors/sql-introspection-connector/src/prisma_1_defaults.rs

```diff
@@ -0,0 +1,97 @@
+use crate::commenting_out_guardrails::ModelAndField;
+use datamodel::{dml, Datamodel, ValueGenerator};
+use introspection_connector::{Version, Warning};
+use quaint::connector::SqlFamily;
+use sql_schema_describer::SqlSchema;
+
+pub fn add_prisma_1_id_defaults(
+    family: &SqlFamily,
+    version: &Version,
+    data_model: &mut Datamodel,
+    schema: &SqlSchema,
+    warnings: &mut Vec<Warning>,
+) {
+    let mut needs_to_be_changed = vec![];
+
+    let varchar = "varchar";
+    let character_varying = "character varying";
+    let varchar_25 = "varchar(25)";
+    let varchar_36 = "varchar(36)";
```

Comment:
So P1 was doing the same as now, using the string representation - binary column support is probably going to be a noticeable performance improvement.

Reply:
Nitpick: these could be `const` items outside of the function just to make it shorter.